ocaml/lib/pickles_bindings.ml

```diff
@@ -620,4 +787,15 @@ let pickles =
       fun (proof : proof) ->
         proof |> Pickles.Side_loaded.Proof.of_proof
         |> Pickles.Side_loaded.Proof.to_base64 |> Js.string
+
+    val encodeVerificationKey = encode_verification_key
+
+    val decodeVerificationKey = decode_verification_key
+
+    val util =
```

Comment:
Do you use this?

Reply:
yes, in one place, here: https://github.com/o1-labs/o1js/pull/1187/files#diff-a0ca705d913f22307bcab0d9758200d95493afcd46279320867493dada7bce68R56